src/Components/Workspace/index.js

```diff
@@ -396,10 +396,12 @@ const WorkspaceView = props => {
           <h2>Project: &quot;{title}&quot;</h2>
         </Col>
       </Row>
-      <Row>
-        <Col sm={ 4 }><h5>Title</h5></Col>
-        <Col sm={ 4 }><h5>Created / Updated</h5></Col>
-        <Col sm={ 4 }><h5>Transcripts</h5></Col>
+      <Row style={ { color: '#6b6b6b' } }>
+        <Col sm={ 8 } style={ { display: 'flex', justifyContent: 'space-around' } }>
+          <h5 style={ { fontWeight: 'normal', borderBottom: 'solid 1px #c0c0c0', paddingBottom: '8px', width: '50%' } }>Programme scripts</h5>
```

Comment:
Just a note to say I know this project uses inline styling but I'm wondering whether it would be nicer to move it to using stylesheets, or would that make things too inconsistent? Conscious that we're adding more here than perhaps we originally thought. Maybe a tech huddle / question for @tamsingreen and @leihelondon

Reply:
☝️ will refactor to Sass